utils/src/test/kotlin/RedirectionTest.kt

```diff
@@ -81,7 +82,7 @@ class RedirectionTest : WordSpec({
             val stdout = redirectStdout {
                 exitCode = trapSystemExitCall {
                     for (i in 1..numberOfLines) System.out.println("stdout: $i")
-                    System.exit(42)
+                    exitProcess(42)
```

Comment:
I'd actually prefer this, like also the other calls in this file, to stick to Java's `System.*`. Here, my main reason is that the fact that `exitProcess()` calls `System.exit()` is an implementation detail. But the trap works for `System.exit()` *only*, so IMO we should be explicit about calling it.

Reply:
Interresting, thanks.